doc/source/contributor/package-updates.rst

```diff
@@ -0,0 +1,202 @@
+=============================
+Package and Container Updates
+=============================
+
+Preparations
+============
+
+1. Before building images, you should check for any outstanding PRs into the “base branch“. Below are the links for the Wallaby branches.
+
+ kayobe-config: https://github.com/stackhpc/stackhpc-kayobe-config/pulls?q=is%3Apr+is%3Aopen+base%3Astackhpc%2Fwallaby
+
+ kolla: https://github.com/stackhpc/kolla/pulls?q=is%3Apr+is%3Aopen+base%3Astackhpc%2Fwallaby
+
+ kolla-ansible: https://github.com/stackhpc/kolla-ansible/pulls?q=is%3Apr+is%3Aopen+base%3Astackhpc%2Fwallaby
+
+ You should also check any referenced source trees in etc/kayobe/kolla.yml.
+
+ e.g: https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/wallaby/etc/kayobe/kolla.yml#L112-L158
+
+2. Follow the workflows documented `here <https://stackhpc.github.io/stackhpc-release-train/usage/content-howto/#update-package-repositories>`_. Sync the package repositories. Then, for each release: update the Kayobe package repository versions, build and push Kolla container images, open a draft PR with the updated container image tags. The rest of this document describes the stage "Test".
+
+Testing
+=======
+
+The following steps describe the process to test the new package and container repositories. See the subsections beneath for further explanations.
+
+1. Build two multinode environments for OVS and OVN, both on the "base branch".
+
+2. Run tests on current package versions as a baseline.
+
+3. Upgrade host packages.
+
+4. Upgrade containers.
+
+5. Run tests again with the new packages.
+
+6. Request reviews for your proposed PR to bring in the new packages.
+
+7. Promote these packages before merging the PR, but after the CI checks have passed.
+
+8. Upgrade OpenStack to the next release.
+
+9.  Repeat steps 2 and 4-7. (Step 3 is skipped as the host packages will be shared across these releases.)
```

Comment:
Need to mention somewhere that you need to start on the earliest supported release.